sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,174 @@
+- Add support for local accessors to lsc API [9dce2d227b31fdbba09a044500a0ea849cdeb747]
+- Replace USM allocator with UMF abstractions in Level Zero backend [7b9b8ce4e4c5be5503b22e4cedf341b4120d1ad6]
+- Make SYCL-Graph functions thread-safe [c8c64a673eebfede0d9dbc0728a4eaaa5bde0b10]
+- Improve diagnostic for nd_item passed to parallel_for with range [e600d7922489635290fd2cb63300d865fc72ee72]
+- Implement missing exceptions defined by SYCL-Graphs specification [77b794bcaad6a33ff5734d938412f61b6b7475f0]
+- Implemented supported make_device, make_queue, make_event interop functions for HIP backend [5e9d07b14a47c2097764b148c3b1dcdb9fcbbc1c]
+- Add missing nearbyint for CUDA [0ef26d3ee8086cb51d4f5deb9ad8c3d81cdd8f16]
+- Implement runtime query for matrix combinations [a09491f4e389dbef29128bda69a7954e4b887af6]
+- Support basic kernel launches over 64-bit global ranges [192c3016e76b36b755bd13b1d32446071812c600]
+- Clarify runtime error for nested kernel submissions [7afc2d03fd0040177009967aca057cc80708747b]
+- Improve handling of spec constants with padding in the middle of structs [292309c826a3e11b8cb225667b316d8ce01e2779]
+- Add support for CUDA backend with graph extension [367b662a49baad1a881696448399d68f491452e4]
+- Add support for specialization constants with graph extension [fca7f5e2e3fb558e613c5daa0eb547b8e25cc524]
+- Improve `vec::convert` implementation to use a single instruction instead of per-element [f634cc9bf799d92029da2b3090db4563191c1f0a]
+
+### Documentation
+- Update sycl_ext_oneapi_local_memory [457251feceb81845de962383f227e14522d4cf96]
+- Add get-coord API and general query example to `sycl_ext_intel_matric` [579f1dec0b5b2ecd177dfb95c3d1522f57079cc6] [38ac2124c1e8034c37056256f9864e475128ad49]
+- Update design doc for new offloading model [115808f5128f33321c68b9dead4fcd25f6a2ed49]
+- Update immediate command list usage [ad973f3383e265a1e209b61559e24cec37c5d111]
+- Add overloads with sycl::queue for some functions in `sycl_ext_oneapi_bindless_images` [4d0c552985bd0e4fc16740a0fc89393ef7fe3d04]
+- Add queue requirement when replay a graph [2c53815c6c1e3f8e40c5b6dbe3e5dc2aebd4fa37]
+
+## Bug Fixes
+### SYCL Compiler
+- Fix backend option passing issues with multiple targets [12ca2db0e30b155c7d6bf14320dd1ba468ac9294]
+- Fix -fgpu-rdc option for CUDA [f7595ac7527e67a59e33550851366db7d886ac3f]
+- Fix triple in offload mismatch warning [0fd9a4eb17a2db0fed151ba9d975804322966bcc]
+- Fix a bug when -enable-global-offset=false for AMDGU and NVPTX targets [00cf4c29740b2ec8d027e59e49e376a211599a58]
+
+### SYCL Library
+- Fix stack corruption in sycl::modf for sycl::half type [7c6541dc18e324e36853b473a15e8df4edf97755]
+- Fix nextafter with half on host [1f10f3588e8ccab79e42eee7fd640153d701b08c]
+- Fix identity values for (sub)group collectives min/max [2483893719324831eebce8c188aed688a8a04419]
+- Improve handling of large reqd_work_group_size values [25c3666dffa935c64db513d0db25205e0400c6b7]
```

Comment:
Corrected exception thrown when kernel is submitted with a large (unsupported) `reqd_work_group_size` attribute